RealmSwift/SwiftUI.swift

```diff
@@ -343,9 +343,9 @@ private class ObservableStorage<ObservedType>: ObservableObject where ObservedTy
     public init(_ type: ResultType.Type,
                 configuration: Realm.Configuration? = nil,
                 filter: NSPredicate? = nil,
-                keyPaths: [String]? = nil,
+                excludedkeyPath: [String]? = nil,
```

Comment:
I'm confused about this change. It seems wrong? Aren't these meant to be specifically the keyPaths you are observing?

Reply:
I think this happened when I was renaming a test variable called "keyPaths" and it included actual variables.